development_page.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <!-- Javascript and CSS links-->
+    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
+        integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
+        crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+        integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+        crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.4.1/dist/js/bootstrap.min.js"
+        integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
+        crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
+        integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+    <link rel="stylesheet" href="development_page_style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css">
+
+    <!-- Character encoding and viewport-->
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- Title for page-->
+    <title>Development</title>
+</head>
+
+<body class="bgimg">
+    <!-- Perfect dark logo-->
+    <div class="text-center p-3 pt-4">
+        <a href="https://en.wikipedia.org/wiki/Perfect_Dark">
+            <img src="https://www.theinitiative.com/wp-content/uploads/2021/07/pd-logo-white.png" width="200"
+                height="60">
+        </a>
+    </div>
+    <header class="sticky-top">
+        <!-- Nav bar -->
+        <nav class="navbar navbar-expand-sm navbar-dark p-3">
+            <div class="collapse navbar-collapse pb-2" id="navbarNav">
+                <ul class="navbar-nav mx-auto">
+                    <li class="nav-item px-3">
+                        <a class="nav-link" href="#">HOME</a>
+                    </li>
+                    <li class="nav-item px-3">
+                        <a class="nav-link" href="#">GAMEPLAY</a>
+                    </li>
+                    <li class="nav-item px-3 active pdfont">
+                        <a class="nav-link" href="#">DEVELOPMENT</a>
```

Comment:
```suggestion
                        <a class="nav-link" href="development_page.html">DEVELOPMENT</a>
```
Maybe also make the href direct to the development page on the navbar since the HTML has been made for it.

Reply:
Yes ur right